src/app/blog/[title]/page.tsx

```
@@ -32,12 +45,99 @@ export default async function Page({ params }: { params: { title: string } }) {
+  if (!post) return;
+
   return (
     <HydrationBoundary state={dehydrate(queryClient)}>
       <div className="relative mx-auto flex size-full flex-col justify-between px-5 py-8 text-lg tablet:w-tablet tablet:px-0">
-        <Suspense>
-          <ClientPage title={title} />
-        </Suspense>
+        <PostHeader post={post} user={user} />
+
+        {/* 목차 */}
+        {post.content && post.content.trim() && <GenerateTOC content={post.content} />}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)